### PR TITLE
Tereses Portfolio

### PR DESCRIPTION
**Pull request by TereseClaesson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
...
\ No newline at end of file
+# portfolio
+
+My portfolio! There is still a lot to be done. I will keep building on this ground from now on.
+
+This weeks projects was to follow a figma design on how to create a mobile and desktop version of a portfolio, that I can use in my job searching in the career coaching. 
+
+This weeks project was also to use cross browers testing and make the webside more web acessible for people with different disabilities.  
+
+# to view
```
Comment (TereseClaesson) [inline README.md:9]: https://tereses-portfolio.netlify.app/

File: index.html [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+        </div> 
+          <img class="profile-picture" src="img/profile pic.jpg" alt="Profile pictures of Terese">
+        </div>
+    </header>
+    <section class="section">
+      <div class="container">
+        <p class="summary-section">
+          <span class="line"></span>
+          <span class="dot"></span>
+        Hello! I am a strategic,flexible and sociable person with experience in the area of biomedical science and knowledge and skills in leadership,quality,improvement and now also skills as a frontend deleloper.I am all about solving problems that matter and creating new meningful solutions.
```
Comment (Laura-Sjolander) [inline index.html]: Looks like there is some < > missing, for the  text startin with "Hello..." so the text doesn´t get the styling.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+         <button>CSS3</button>
+        </div>
+        </div>  
+      </div>
+    </div>  
+  </article>
+        <div class="project-text">
+         <h2 class="project-heading">Other projects</h2>
+        <span class="highlight">Ski Lessons</span><br>
+          <a href="https://github.com/TereseClaesson/project-business-site" tabindex="0" role="button" aria-pressed="false" aria-label="business site repository" target="_blank" rel="noreferrer">
```
Comment (Laura-Sjolander) [inline index.html]: Nice that you got the aria-roles in here for accessability purposes!

File: index.html [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer class="footer">
+      <div class="contact">
+        <p>Contact</p>
+        <p>Terese Claesson</p>
+        <p>0703-270796</p>
+        <p><a href="mailto:terese.claesson@gmail.com">terese.claesson@gmail.com</a></p>
```
Comment (Laura-Sjolander) [inline index.html]: Great that this is a mailto-link, so you get it up directly in your e-mail as a new e-mail draft!

File: style.css [hunk 5]
```diff
@@ -0,0 +1,509 @@
...
+  width: 375px;
+  margin: 0 auto;
+  padding: 40px 40px;
+}
+
+.section:nth-child(2n) {
+  background-color: #f4f4f4;
+}
+.section:nth-child(2n + 1) {
+  background-color: #fff;
```
Comment (Laura-Sjolander) [inline style.css]: Great using nth-child for the different backgroundcolours.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,509 @@
...
+
+.color {
+  color: #db9aab;
+}
+
+.tech-title {
+  font-family: "Montserrat", sans-serif;
+}
+
+.tech-text {
```
Comment (Laura-Sjolander) [inline style.css]: I really liked how you have named the classes, it´s easy to follow along when the classname tells me what it is you have styled!

**Review by Laura-Sjolander — COMMENTED** [review]
I really liked your portfolio, as well as the text and pictures chosen. 

It is a very clean and well structured code

It´s easy to follow along since you named sections and classes descriptive- it tells the reader what it is you have in the section or what you are styling and so on.

You have worked with the accessability part with ARIA roles and so on. Good job! 

Looks nice.   You should be really proud of yourself!

**Pull request closed without merging**